api/src/services/eml-service.ts

```diff
@@ -666,44 +666,36 @@ export class EmlService extends DBService {
   /**
    * Creates an object representing the dataset creator from the given projectData.
    *
+   *
    * @param {IGetProject} projectData
    * @return {*}  {Record<string, any>}
    * @memberof EmlService
    */
   _getProjectDatasetCreator(projectData: IGetProject): Record<string, any> {
-    const primaryContact = projectData.coordinator;
-
-    if (JSON.parse(primaryContact.share_contact_details)) {
-      // return full details of the primary contact iff it is public.
-      return {
-        organizationName: primaryContact.coordinator_agency,
-        electronicMailAddress: primaryContact.email_address
-      };
-    }
+    const participant = projectData.participants[0];
```

Comment:
In the few places that grab the 0th element, would you mind changing it to instead find a participant with the coordinator role? It should always exist, though I guess the find won't know that.. Might have to bring that if else structure back from the old version, just to appease the type safety.

Reply:
(It probably doesn't really matter in the end, but we've come this far :/)

Reply:
Good call. Changed!